ch.sbb.scion.rcp.microfrontend/src/ch/sbb/scion/rcp/microfrontend/browser/JavaCallback.java

```diff
@@ -20,15 +19,15 @@ public class JavaCallback implements IDisposable {
 
   public final String name;
 
-  private CompletableFuture<Browser> whenBrowser;
-  private BrowserFunction browserFunction;
+  private CompletableFuture<BrowserView> whenBrowser;
   private Consumer<Object[]> callback;
+  private DisposableJsFunction browserFunction;
 
-  public JavaCallback(Browser browser, Consumer<Object[]> callback) {
+  public JavaCallback(BrowserView browser, Consumer<Object[]> callback) {

```

Comment:
Let's either follow the pattern on declaring the arguments as `final` everywhere, or skip it everywhere.